lib/ui/form/form_screen.dart

```diff
@@ -105,6 +123,37 @@ class FormScreenState extends ConsumerState<FormScreen> {
                       question: questions[index - 1],
                       questionIndex: index,
                       questionTotal: questionTotal,
+                      onSelectedIndexedAnswers: (selectedIndexes) {
+                        final ansOption = questions[index - 1]
```

Comment:
What about this?

```suggestion
                        final answerIds = questions[index - 1]
```

Reply:
Instead of fetching the answer ID in `form_screen`, can we get the answer ID from the widget instead?

(Ex: Similar what you are doing for `form_survey_answer_text_field.dart`)